### PR TITLE
feat(MembershipsAdapter): add the boolean property "host" for the Member object

### PR DESCRIPTION
**Pull request by karinasigartau0798**
Add the host property for the Member object.

### CODE REVIEW

File: src/MembershipsAdapter.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import WebexAdapter from './WebexAdapter';
  * @property {boolean}  muted       If the member's audio is currently muted
  * @property {boolean}  sharing     If the member is currently sharing their screen
  * @property {boolean}  inMeeting   If the member is currently in meeting
+ * @property {boolean}  hosting     If the member is the host of a meeting
```
Comment (lalli-flores) [inline src/MembershipsAdapter.js:14]: Being a host is a role in a meeting, not an action so we should name it accordingly.
```suggestion
 * @property {boolean}  host     If the member is the host of a meeting
```

**Review by lalli-flores — COMMENTED** [review]
Let's call the attribute "host" as that is the role name. Don't forget to update the commit message accordingly

File: src/MembershipsAdapter.js (hunk 1 above)
Comment (karinasigartau0798) [reply src/MembershipsAdapter.js]: Done.

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**